configs/condinst/condinst_r50_fpn_ms_1x_coco.py

```diff
@@ -0,0 +1,110 @@
+_base_ = [
```

Comment:
You can also add a config based on the detectron2's 90k schedule in configs/common/ms-poly-90k_coco-instance.py

Reply:
Thanks for your kind suggestion. I have tried the 90k shcedule and i got a precision:
MS_R_50_1x| box AP | mask AP |
--- |:---:|:---:|
origin   | 39.7 | 35.7 | 
ours| 39.9 | 35.5 |

Is a AP error of 0.2% acceptable?

Reply:
> Great! I think an error of 0.2% is acceptable.

Thanks! 
I have made corrections based on your comments, can you please review it again?